src/sinks/greptimedb/metrics/config.rs

```diff
@@ -0,0 +1,127 @@
+use vector_lib::configurable::configurable_component;
+use vector_lib::sensitive_string::SensitiveString;
+
+use super::request::GreptimeDBGrpcRetryLogic;
+
+use super::service::{healthcheck, GreptimeDBGrpcService};
+use super::sink;
+use crate::sinks::greptimedb::{default_dbname, GreptimeDBDefaultBatchSettings};
+use crate::sinks::prelude::*;
+
+/// Configuration items for GreptimeDB
+#[configurable_component(sink("greptimedb", "Ingest metrics data into GreptimeDB."))]
```

Comment:
Because we now have a `greptimedb_logs` sink we will need to rename this to `greptimedb_metrics`. We will need to maintain an alias from `greptimedb` to `greptimedb_metrics`. 

The `mezmo` formerly `logdna` sink is a good example of where we have done this before.

See the code [here](https://github.com/vectordotdev/vector/blob/master/src/sinks/mezmo.rs#L29-L34). We should alias in the [docs](https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sinks/mezmo.cue#L5)


```suggestion
#[configurable_component(sink("greptimedb_metrics", "Ingest metrics data into GreptimeDB."))]
```